src/Stratis.Bitcoin.Tests.Wallet.Common/WalletTestsHelpers.cs

```diff
@@ -556,19 +588,24 @@ public void Dispose()
         /// <param name="name">The name.</param>
         /// <param name="password">The password.</param>
         /// <returns>The generated wallet.</returns>
-        public Features.Wallet.Wallet GenerateBlankWallet(string name, string password)
+        public Features.Wallet.Wallet GenerateBlankWallet(string name, string password, IWalletRepository walletRepository = null)
         {
+            /*
```

Comment:
Commented?

Reply:
The code attempts to buffer and clone wallets. This was causing issues so I've now remove it.